builders/interoperability/xcm/send-execute-xcm.md

```diff
@@ -173,9 +173,30 @@ Polkadot XCM Pallet的`execute`函数接受两个参数：`message`和`maxWeight
 
 1. 创建提供商和签署者
 2. 创建用于交互的XCM Utilities Precompile的实例
-3. 定义`xcmExecute`函数所需的参数，这些参数将是XCM消息的编码调用数据以及用于执行消息的最大权重。您可以将`maxWeight`设置为`100000000000n`，它对应于`refTime`。`proofSize`将自动设置为默认值，即64KB
+3. 定义`xcmExecute`函数所需的参数，这些参数将是XCM消息的编码调用数据以及用于执行消息的最大权重。您可以将`maxWeight`设置为`400000000n`，它对应于`refTime`。`proofSize`将自动设置为默认值，即64KB
 4. 执行XCM消息
 
+!!! 请记住
```

Comment:
This part was left out and I felt like the best way to handle it was to add in the code snippets for now, and then there is a ticket open to translate changes to this page so the exact translations will be handled later so it aligns 100% with the EN repo

Reply:
ref: https://github.com/moonbeam-foundation/moonbeam-docs/pull/724/files#diff-9668549d3ea32ff3d454a3fcb3a8822badb767e96fb84850761519744bb2016f